src/lib/users/ssh_authorized_keyring.rb

```diff
@@ -62,26 +62,34 @@ def default_message
 
       # The home directory does not exist.
       class HomeDoesNotExist < PathError
-        # @return default_message [String] Default error message
-        def default_message; "Home directory does not exist" end
+        # @return [String] Default error message
+        def default_message
+          "Home directory does not exist"
```

Comment:
no translation? Do we show it to users or not?

Reply:
No, we just log this information. See https://github.com/yast/yast-users/blob/a90550d5b483982b93aa88fc7dafa9f24fd96f3d/src/modules/SSHAuthorizedKeys.rb#L51.